tensorflow_datasets/image_classification/places365_small.py

```diff
@@ -134,6 +134,7 @@ def _generate_examples(self, archive, path_prefix, split_name,
         file_to_class = {x[0]: int(x[1]) for x in csv.reader(f, delimiter=" ")}
 
     for fname, fobj in archive:
+      fname = fname.replace("\\", "/")
       assert fname.startswith(path_prefix)
```

Comment:
I'm confused how can the test pass if `path_prefix` is created with `os.path.join` (so contains the '\\'), but `fname.replace` contains the '/' ?

Reply:
From what I understand,
`path_prefix` is a string constant like `test_256`, `val_256` and from the code, it does not look like it is created by `os.path.join`.

Reply:
Yes, indeed, I misread the code. Sorry.